Request/RequestBodyParamConverter20.php

```diff
@@ -16,7 +16,7 @@
 
 /**
  * This code is needed for SensioFrameworkExtraBundle 2.x compatibility
- * https://github.com/FriendsOfSymfony/FOSRestBundle/issues/622
+ * https://github.com/FriendsOfSymfony/FOSRestBundle/issues/622.
```

Comment:
yeah .. we will keep sensio 2.x support for now which is why I was saying please remove the above change.